tests/utils/test_validation.py

```diff
@@ -43,6 +46,13 @@
 ]
 
 
+def test_path_not_unique():
+    for unique_path in UNIQUE_PATHS:
+        assert not path_not_unique(unique_path)
+    for not_unique_path in NOT_UNIQUE_PATHS:
+        assert path_not_unique(not_unique_path)
```

Comment:
You can use `pytest.mark.parametrize` here.

```python

@pytest.mark.parametrize(
    ("path", "expected"),
    [
        ("a", True),
        ...
    ]
)
def test_path_not_unique(path, expected):
    assert path_not_unique(path) is expected
```

Reply:
I applied your FB! https://github.com/mlflow/mlflow/pull/7447/commits/869bc324b3bc2e3fbee00bf937e7e727803bb41a

Reply:
how do I re run the failed job https://github.com/mlflow/mlflow/actions/runs/3616870043/jobs/6095235587 ?

Reply:
Only maintainers can rerun jobs. I reran it.